syft/serde/serde.py

```diff
@@ -329,10 +334,13 @@ def _compress(decompressed_input_bin: bin) -> bin:
 
     """
     compress_stream, compress_scheme = _apply_compress_scheme(decompressed_input_bin)
-    if len(compress_stream) < len(decompressed_input_bin):
-        return compress_scheme.to_bytes(1, byteorder="big") + compress_stream
-    else:
-        return NO_COMPRESSION.to_bytes(1, byteorder="big") + decompressed_input_bin
+    try:
```

Comment:
Same question as above

Reply:
Same answer ;) This should work in 99.99% of the casee